source/manual/data-gov-uk-supporting-ckan.html.md

```diff
@@ -424,3 +424,32 @@ running, so is safe to run immediately if you suspect the process has crashed:
 ```bash
 $ fab aws_production class:ckan ckan.restart_harvester
 ```
+
+### CKAN publisher on Staging environment responds with Nginx 504 timeout:
+
+Sometimes CKAN publisher on Staging environemt responds with a 504 from Nginx, this is due to it timing out when connecting to the database as there are too many connections, current limit is 1000.
```

Comment:
Might be good to mention that it's only supposed to need one connection when it's working normally.